pages/rdm_cycle/collecting.md

```diff
@@ -1,41 +1,45 @@
 ---
 title: Collecting
 keywords:
-contributors: [Korbinian Bösl, Siiri Fuchs, Anastasia Chasapi]
+contributors: [Korbinian Bösl, Siiri Fuchs, Anastasia Chasapi, Ulrike Wittig]
 ---
 
 ## What is Data Collection?
 
-Data collection is the process where information is gathered about specific variables of interest either using instrumentation or other methods. While data collection methods depend on the field and research subject, it is important to ensure data quality.
+Data collection is the process where information is gathered about specific variables of interest either using instrumentation or other methods (e.g. questionnaires, patient records). While data collection methods depend on the field and research subject, it is important to ensure data quality.
+
+You can also reuse existing data in your project. This can either be individual earlier collected datasets or reference data from curated resources like [ELIXIR Core Data Resources](https://elixir-europe.org/platforms/data/core-data-resources) or consensus data like reference genomes.  For more information see [Reuse](reusing) in the data life cycle. 
 
-You can also reuse existing data in your project, for more information see “Reuse” in the data life cycle.
 
 ## Why is Data Collection important?
 
 Apart from being the source of information to build your findings on, the collection phase lays the foundation for the quality of both the data and its documentation. It is important that the decisions made regarding quality measures are implemented, and that the collect procedures are appropriately recorded. 
 
+
 ## What should be considered for Data Collection?
 
 Consider the following, while collecting data
 
 * How to capture provenance - e.g. of samples, researchers and instruments
-* Ensure data quality
+* Ensure data quality - data can either be generated by yourself, or by another infrastructure or facility with this specialization
+* Reusing data instead of generating new data
 * Experimental design - including a collection plan (e.g. repetitions, controls, randomization) in advance
 * Instrument calibration
 * If you work with sensitive or confidential data, take care of data protection and security issues
 * If you work with human-related data, think about permissions, consent
-* How will you store the data?
-* Where will you store the data?
-* Describing the data, is there a suitable metadata standard?
+* How to store the data
+* Where to store the data
+* Identify suitable metadata standards
 
 
-## Where can training materials and events about Data Collection be found?
+## Problems to be addressed in this stage
 
-{% include tess.html search="Data Collection" %}
+{% include pagelist.html tag="collect" %}
 
-## Related topics
 
-{% include pagelist.html tag="collect" %}
+## Available training material
```

Comment:
The title of the paragraph about training is still: Where can training materials and events about Data Sharing be found? or WP2 wants to change that?